alcs-api/src/comment/comment.service.ts

```diff
@@ -83,36 +84,41 @@ export class CommentService {
   async update(uuid: string, body: string, mentions: CommentMention[]) {
     const comment = await this.commentRepository.findOne({
       where: { uuid },
-      relations: { ...this.DEFAULT_COMMENT_RELATIONS, application: true },
+      relations: {
+        ...this.DEFAULT_COMMENT_RELATIONS,
+      },
     });
 
     if (body.trim() === '') {
       throw new ServiceValidationException('Comment body must be filled.');
     }
 
-    if (comment.body.trim() === body.trim()) {
```

Comment:
Why is this check being removed?

Reply:
initially, this check was added to avoid errors on entity updates if nothing changed. That was caused with date transformers that were addressed previously so this one is redundant